content/api/suppression-list.apib

```diff
@@ -254,16 +238,6 @@ If the X-MSYS-SUBACCOUNT header is not provided, a search will be performed acro
             Accept: application/json
             X-MSYS-SUBACCOUNT: <subaccount-id> (optional)
```

Comment:
Remove this. It does nothing with the new template and isn't a good way to document this anyways.